### DISCUSSION TITLE
Do ranked scores,achievements & exp gain work in osu lazer ?

### DISCUSSION THREAD

**Discussion by mdrokz** [ppy/osu#22702] [Q&A]
Hi im not sure if ranked score or exp gain works in osu lazer because i played alot of ranked beatmaps and i havent leveled up or gotten any rank score, and i got alot of A and S on ranked beatmaps but it doesnt show on my profile ? can anyone help me please ? thanks

**Comment by bdach** [comment]
Ranked score and experience should work, albeit only on the experimental website instance (https://lazer.ppy.sh/home). Not entirely sure about medals. I think beatmap pack ones may do but others not?

**Reply by Axilotl17** [reply]
I was literally typing a very similar response but you beat me to it by a few seconds, haha!

**Reply by mdrokz** [reply]
> Ranked score and experience should work, albeit only on the experimental website instance (https://lazer.ppy.sh/home). Not entirely sure about medals. I think beatmap pack ones may do but others not?

![Screenshot from 2023-02-21 02-10-50](https://user-images.githubusercontent.com/33598169/220345019-268b0e13-7f4b-4c8e-9a59-3e57bdc530ff.png)

https://lazer.ppy.sh/users/26153450 i tried alot of ranked beatmaps but it doesnt work for me

**Reply by Axilotl17** [reply]
are you logged in? can you play a map and then send the logs here?

**Reply by mdrokz** [reply]
> are you logged in? can you play a map and then send the logs here?

Yes i login and play, let me try sending logs here

**Reply by mdrokz** [reply]
> are you logged in? can you play a map and then send the logs here?

here [logs.zip](https://github.com/ppy/osu/files/10796481/logs.zip)

**Comment by ItsShamed** [comment]
Yes ranked scores and leveling works on the https://lazer.ppy.sh instance. Looking at your profile, you have scores on your [osu!catch](https://lazer.ppy.sh/users/26153450/fruits) and [osu!mania](https://lazer.ppy.sh/users/26153450/mania) profile. 
For medals see #18880

In https://github.com/ppy/osu/discussions/22702#discussioncomment-5063594 you are most likely seeing your osu!std profile, which is marked as your default profile (and has currently no scores on it). Some of the scores you submitted (seen in your attached logs) were not calculated yet, but still displayed.

**Reply by mdrokz** [reply]
> Yes ranked scores and leveling works on the https://lazer.ppy.sh instance. Looking at your profile, you have scores on your [osu!catch](https://lazer.ppy.sh/users/26153450/fruits) and [osu!mania](https://lazer.ppy.sh/users/26153450/mania) profile. For medals see #18880
> 
> In [#22702 (reply in thread)](https://github.com/ppy/osu/discussions/22702#discussioncomment-5063594) you are most likely seeing your osu!std profile, which is marked as your default profile (and has currently no scores on it). Some of the scores you submitted (seen in your attached logs) were not calculated yet, but still displayed.

ah i see now it makes sense i didnt notice that there were different profiles for the game modes, also one more question i dont get any beatmap rank awards even though i have alot of S and A ranked beatmaps played and also 1st place awards

![image](https://user-images.githubusercontent.com/33598169/220666980-ddb03f5a-5e79-48f4-a761-cf70c93209a2.png)


![image](https://user-images.githubusercontent.com/33598169/220667054-d543b729-bc88-41bf-be2c-0e2f603478a2.png)

**Reply by ItsShamed** [reply]
It seems that it's still tracking the values from the https://osu.ppy.sh instance. Not sure if this issue is known though.

**Reply by Axilotl17** [reply]
this is known. https://lazer.ppy.sh is a combination of lazer and stable scores.

**Reply by ItsShamed** [reply]
Sure but is it expected to not aggregate the graded score counts (only counts stable ones at the moment)?

**Reply by mdrokz** [reply]
> Sure but is it expected to not aggregate the graded score counts (only counts stable ones at the moment)?

Is there any solution to this ?

**Reply by rvnexe** [reply]
Same here!

**Reply by ItsShamed** [reply]
This might probably need an issue on https://github.com/ppy/osu-web

**Comment by rvnexe** [comment]
I have this problem too ( #22838 )
